SEQLinkage/Runner.py

```diff
@@ -139,7 +142,7 @@ def format_merlin(tped, tfam):
 #because the haplotype patterns are different from family to family.
 #You can analyze them all together
 def format_linkage(tped, tfam, prev, wild_pen, muta_pen, inherit_mode, theta_max, theta_inc):
-    out_base = '{}/LINKAGE/{}'.format(env.tmp_dir, splitext(basename(tped))[0])
+    out_base = '{}/LINKAGE/{}'.format(env.outdir, splitext(basename(tped))[0])
```

Comment:
Please double-check this. I think we distinguished between temp and out dir for a reason: we can safely remove anything in temp dir and only keep the out dir at the end of the analysis.

Reply:
This is the linkage format output.